pages/oss.html

```diff
@@ -25,7 +25,7 @@
 
 <ol>
     <li>
-    <p>We are working with internal offices that develop custom code and explaining the <a href="https://sourcecode.cio.gov/" target="blank">M-16-21</a> and our <a href="{{site.baseurl}}/assets/files/GSAOSSPolicy.pdf" target="_blank">responsibilities</a>.  This includes creating guidance for inventorying and for opening code.
+    <p>We are working with internal offices that develop custom code and explaining the <a href="https://web.archive.org/web/20170121010212/https://sourcecode.cio.gov/" target="blank">M-16-21</a> and our <a href="{{site.baseurl}}/assets/files/GSAOSSPolicy.pdf" target="_blank">responsibilities</a>.  This includes creating guidance for inventorying and for opening code.
```

Comment:
We'd like to use .gov sources whenever possible. If you can update your PRs to use [this link](https://obamawhitehouse.archives.gov/sites/default/files/omb/memoranda/2016/m_16_21.pdf), we'd be happy to merge them.